CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing Guidelines
+
+The following is a set of guidelines for contributing to the NGINX Ingress Controller. We really appreciate that you are considering contributing!
+
+
+
+## Table of Contents
+
+- [Contributing](#contributing)
+  - [Types of Contributions](#types-of-contributions)
+  - [Get Started](#get-started)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Tips](#tips)
+
+
+## Contributing
+
+Contributions are welcome, and they are greatly appreciated\! Every
+little bit helps, and credit will always be given.
```

Comment:
You probably don't need the "Contributing" heading since this is the Contribution Guidelines page.